presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/MetadataDao.java

```diff
@@ -55,7 +55,7 @@ TableColumn getTableColumn(
             @Bind("columnId") long columnId);
 
     @SqlQuery("SELECT t.schema_name, t.table_name,\n" +
-            "  c.column_id, c.column_name, c.ordinal_position, c.data_type\n" +
+            "  c.column_id, c.column_name, c.ordinal_position, c.data_type, c.bucket_ordinal_position, c.sort_ordinal_position, t.temporal_column_id = c.column_id AS temporal\n" +
```

Comment:
I'm thinking we should get rid of duplicate method (the two usages can just grab all columns), or if it really mattered, use the `(column_id = :columnId OR :columnId IS NULL)` trick.